doc/source/user_guide/index.rst

```diff
@@ -16,6 +16,6 @@ the package, see the :ref:`pysteps-reference`.
     install_pysteps
     example_data
     set_pystepsrc
-    ../auto_examples/index
+    ../examples_gallery/index
```

Comment:
so the url to the example gallery will change, correct? Then I think there are few more places (e.g. README, github page) where we need to update links.